### ISSUE TITLE
Redux Integration: fails on V6 

### ISSUE THREAD

**Issue by dilip025** [nozzle/react-static#960]
![image](https://user-images.githubusercontent.com/45608461/50991315-60296200-14e2-11e9-8078-65fe4674c638.png)

**Comment by tannerlinsley** [comment]
Initially, it looks like an issue with the redux package export. Did you recently upgrade versions of redux?

**Comment by dilip025** [comment]
Although I am getting errors, seems like build is fine. I am getting the below in the console. 
`{ createStore: [Function: createStore],
  combineReducers: [Function: combineReducers],
  bindActionCreators: [Function: bindActionCreators],
  applyMiddleware: [Function: applyMiddleware],
  compose: [Function: compose],
  __DO_NOT_USE__ActionTypes:
   { INIT: '@@redux/INITq.u.b.4.1.b',
     REPLACE: '@@redux/REPLACE7.u.c.8.5.v',
     PROBE_UNKNOWN_ACTION: [Function: PROBE_UNKNOWN_ACTION] } }`
![image](https://user-images.githubusercontent.com/45608461/50992241-91a32d00-14e4-11e9-9c46-245c3ed2fbdf.png)

Atleast I could serve the local site. But running dev server, I couldn't even do that.

**Comment by tannerlinsley** [comment]
Please post a link to you repo or a place where we can reproduce the issue on our end?

**Comment by dilip025** [comment]
Here you go. https://github.com/dilip025/react-static-migration

**Comment by tannerlinsley** [comment]
Not yet.

**Comment by craighillwood** [comment]
Hey @dilip025 ! I had the same issue and fixed it by changing the filename of `redux/index.js` to `redux/store.js`.

If I am not mistaken, this error is caused by the order of paths in webpack config's `modules` property [dev example](https://github.com/nozzle/react-static/blob/master/packages/react-static/src/static/webpack/webpack.config.dev.js#L47), so webpack tries to find `redux` module when doing `import { ... } from 'redux`, and its first match is inside `src` path (instead of `node_modules`), but can't find the proper `export`s (which are part of the "real" `redux` module) and then throws an error.

@tannerlinsley I think setting `NODE_MODULES` before `SRC` is a safer option, what do u think?

**Comment by craighillwood** [comment]
@tannerlinsley CRA checks for `node_modules` first 👍 
https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L258

**Comment by tannerlinsley** [comment]
Yeah, we'll have to look at doing this change and what implications it has. It could potentially break current builds, but would probably be a good idea.

**Comment by craighillwood** [comment]
@tannerlinsley fixed! 🙌

**Comment by dilip025** [comment]
Great. Thanks @tannerlinsley.
